### PR TITLE
gh-110631: Fix reST indentation in `Doc/library`

### PR DESCRIPTION
**Pull request by ezio-melotti**
This PR fixes indentation errors that resulted in spurious `blockquote`s and definition lists in the `Doc/library` dir.

* Issue: gh-110631

----
:books: Documentation preview :books:: https://cpython-previews--110685.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/tkinter.ttk.rst [hunk 1]
```diff
@@ -986,19 +986,19 @@ ttk.Treeview
...
          The minimum width of the column in pixels. The treeview widget will
          not make the column any smaller than specified by this option when
          the widget is resized or the user drags a column.
-      * stretch: ``True``/``False``
+      stretch: ``True``/``False``
          Specifies whether the column's width should be adjusted when
          the widget is resized.
-      * width: width
+      width: width
          The width of the column in pixels.
```
Comment (ezio-melotti) [inline Doc/library/tkinter.ttk.rst:1002]: This and the next few chunks created a weird definition list with bullets.  I removed the bullets to create a regular definition list.  The markup of the `dt`s could be improved by adding some ` ``...`` ` too, to make them stand out more.

**Comment by erlend-aasland** [comment]
Sphinx indents should be three, not two spaces, no?

**Review by erlend-aasland — APPROVED** [review]

File: Doc/library/sqlite3.rst
Comment (erlend-aasland) [inline Doc/library/sqlite3.rst:1]: How about normalising the `:param:` indents also?

Comment (ezio-melotti) [reply Doc/library/sqlite3.rst:1]: In a separate pull request, sure.

**Review by hugovk — APPROVED** [review]

**Pull request merged**